src/Android/Avalonia.Android/AvaloniaView.cs

```diff
@@ -22,15 +23,35 @@ public class AvaloniaView : FrameLayout
 
         private IDisposable? _timerSubscription;
 
-        public AvaloniaView(Context context) : base(context)
+        // https://learn.microsoft.com/en-us/previous-versions/xamarin/android/internals/architecture#java-activation
+        [System.ComponentModel.Browsable(false)]
+        [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
+        public AvaloniaView(IntPtr javaReference, JniHandleOwnership transfer): base(javaReference, transfer)
```

Comment:
Context itself isn't required in the constructor of AvaloniaView. So technically you can reuse code here.

Reply:
I mean, call original ctor from this one too.

Reply:
As the [documentation ](https://learn.microsoft.com/en-us/previous-versions/xamarin/android/internals/architecture#java-activation) explains. The constructor (IntPtr, JniHandleOwnership) automatically invokes the overload (Context, IAttributeSet, int)

Reply:
Oh, okay